src/UserSpaceInstrumentation/include/UserSpaceInstrumentation/Attach.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
```

Comment:
This file is in the "include" directory, i.e., it is a "public" header of the module. Is it meant to be included (possibly transitively) by modules that depend on this one? For how I would expect it to be an implementation detail and hence be with the private sources, but you have the bigger picture.

Reply:
I'd like to keep this public for now and potentially restrict visibility later. The reason is that I want to use this from LinuxTracing for trying things out. You might be right about this becoming an implementation detail finally.